docs/UserGuide.md

```diff
@@ -44,17 +44,30 @@ Note how the app contains some sample data.<br>
 
 ### Add recipe
 Add a recipe by specifying its name, ingredients, and steps to prepare the dish.<br>
-Usage: `create recipe n/ <name of recipe> i/ <ingredients separated by commas> s/ <steps separated by commas>` <br>
+Usage: `add -n <name of recipe> -d <completion time of recipe> -ss <serving size of recipe> -i <ingredients of recipes separated by commas> -s <steps separated by commas> -t <tags associated with recipe> (optional)` <br>
```

Comment:
I like the usage of angled brackets (<>) to denote mandatory 
Just a nit, I think we should denote the optional parameters in square brackets such as:

```suggestion
Usage: `add -n <name of recipe> -d <completion time of recipe> -ss <serving size of recipe> -i <ingredients of recipes separated by commas> -s <steps separated by commas> [-t <tags associated with recipe>]` <br>
```

We should also mention nearer to the start that the user guide about how to interpret the [usage messages](https://en.wikipedia.org/wiki/Usage_message).

Reply:
I think using (optional) is easier for users to understand tho - q self explanatory and doesn't require users to have to remember convention